gulpfile.js

```diff
@@ -65,9 +65,19 @@ gulp.task('copy-content-assets', () => {
     .pipe(gulp.dest('./dist/en'));
 });
 
+gulp.task('copy-node_modules-assets', () => {
```

Comment:
I was hoping to get rid of gulp at some point but I guess it's still proving useful as a file copy thinger... Though I'm wondering if we could drop it for an npm script or something...

Reply:
I think folks overthink build systems. I've dropped Gulp from a bunch of my projects over the last couple of years in favour of literally just simple Node scripts which copy things into place. (You don't just need to try to do it in bash :) ).